lib/mitmproxy-addon.py

```
@@ -0,0 +1,107 @@
+            "url": flow.request.pretty_url,
+            "status_code": flow.response.status_code,
+            "reason": flow.response.reason,
+            "headers": dict(flow.response.headers),
+            "content_length": len(flow.response.content) if flow.response.content else 0,
+        }
+        
+        # Log response body for small responses
+        if flow.response.content and len(flow.response.content) < 10000:
+            try:
+                response_data["body"] = flow.response.text
+            except:
```

Ruff (0.14.3)

Do not use bare `except`

(E722)